src/styles/settings/index.scss

```diff
@@ -7,13 +7,42 @@ $cdr-warn: false;
 @import './visibility.vars';
 @import './fluid.vars';
 
+// Import fluid.vars.scss if using in a component
```

Comment:
why are these directly in index I would expect that index only imports child files into a central index of containing files?

Reply:
Will do to break apart a bit more

Reply:
Broke this file apart a lot more